install-dependencies.sh

```diff
@@ -125,6 +125,7 @@ pip_packages=(
     scylla-driver
     geomet
     traceback-with-variables
+    scylla-api-client
 )
```

Comment:
I see no call for `pip_packages` on this script (one print only) is it called by another script or we should add it?

Reply:
It's for 729d0feef0f12a13644d8f7c887323df49f53527, we generate PIP package list in --print-pip-runtime-packages, pass the list to tools/python3/reloc/build_reloc.sh via configure.py.

Which is part of 59a6e08cb9a3efb6503e5f3ddc2cb0b784ed27be, bring python module list from scylla-python3 repo to scylla main repo, to make easier maintaining our packages.
(Previously, we needed to update both side to add single python module)